python_bindings/tests/bindings_test.py

```diff
@@ -1,11 +1,13 @@
 import os
 import unittest
 
+import numpy as np
```

Comment:
Dependencies sorted in all Python code according to PEP8 https://www.python.org/dev/peps/pep-0008/#imports

(I ran isort)